src/server/context.ts

```diff
@@ -442,12 +446,14 @@ export class ServerContext {
       const { localUrl, path, size, contentType, etag } of this.#staticFiles
     ) {
       const route = sanitizePathToRegex(path);
-      routes[`GET@${route}`] = this.#staticFileHandler(
```

Comment:
Similarly here, why chance it?

Reply:
I assume Leo changed these to simplify the router (as we don't have to parse `xx@xx` syntax anymore.